pandas/tseries/offsets.py

```diff
@@ -1846,12 +1846,16 @@ def apply(self, other):
         currentEaster = datetime(currentEaster.year, currentEaster.month, currentEaster.day)
         
         # NOTE: easter returns a datetime.date so we have to convert to type of other
-        if other >= currentEaster:
-            new = easter(other.year + self.n)
-        elif other < currentEaster:
-            new = easter(other.year + self.n - 1)
+        if self.n >= 0:
```

Comment:
@sinhrks This checks `self.n >= 0` while pretty much all other offsets check `self.n > 0`.  Is this difference intentional?  If so I'd like to add a brief comment explaining why.

Up-to-date location for this check:
https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/offsets.py#L2129

Examples for other offset classes:
https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/offsets.py#L950
https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/offsets.py#L1409
https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/offsets.py#L1572
https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/offsets.py#L1053